apps/condo/domains/common/components/settings/MobileFeatureConfigContent.tsx

```diff
@@ -0,0 +1,55 @@
+import { Col, Row } from 'antd'
+import { Gutter } from 'antd/es/grid/row'
+import get from 'lodash/get'
+import React from 'react'
+
+import { useFeatureFlags } from '@open-condo/featureflags/FeatureFlagsContext'
+import { useOrganization } from '@open-condo/next/organization'
+
+import { CardsContainer } from '@condo/domains/common/components/Card/CardsContainer'
+import { SettingCardSkeleton } from '@condo/domains/common/components/settings/SettingCard'
+import { MOBILE_FEATURE_CONFIGURATION, TICKET_SUBMITTING_FORM_RESIDENT_MOBILE_APP, SUBMIT_ONLY_PROGRESSION_METER_READINGS } from '@condo/domains/common/constants/featureflags'
+import {
+    OnlyProgressionMeterReadingsSettingCard,
+} from '@condo/domains/settings/components/ticketSubmitting/OnlyProgressionMeterReadingsSettingCard'
+import { TicketSubmittingSettingCard } from '@condo/domains/settings/components/ticketSubmitting/TicketSubmittingSettingCard'
+import { MobileFeatureConfig as MobileFeatureConfigAPI } from '@condo/domains/settings/utils/clientSchema'
+
+const CONTENT_GUTTER: Gutter | [Gutter, Gutter] = [0, 40]
+
+export const MobileFeatureConfigContent: React.FC = () => {
```

Comment:
Confusing naming. It's look like some guard or some wrapper for feature flag checks.

It will be better to create some wraper or HOC for this task. For example:

```
const CardsContainer = ...

const CardsContainerWithAccessCheck = addFeatureFlagIsEnabled(CardsContainer, 'MOBILE_FEATURE')

```